automation_tools/utils.py

```diff
@@ -11,6 +11,17 @@
 from six.moves.urllib.request import urlopen
 
 
+def ansible_setup():
+    with open("ansible_runner/inventory/hosts", "w") as file:
+        for host in env['hosts']:
+            file.write(host + '\n')
+
+
+def ansible_cleanup():
```

Comment:
@JacobCallahan moved writing hosts to inventory file and added removing hosts to here.